content/p/vulnerability-disclosure-policy.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Vulnerability disclosure policy"
+type: p
+---
+
+Do you think you've found a security flaw in AlmaLinux OS or one of our related projects? Read below for how to responsibly report it!
+
+## Reporting a Vulnerability
+
+> - [Our security.txt file](/security.txt).
+
+Taking the time to report a security vulnerability to us is greatly appreciated, and we will use ever resource at our disposal to respect your time during the reporting process. When reporting an issue, please provide **as much information** as possible, but at least:
+
+- The project and version (even better if you can identify the specific commit) where you identified the vulnerability
+- A detailed description of the steps to reproduce 
+- If appropriate, please include a proof of concept (plaintext only; no binaries)
+- Please also include your recommended remediation(s), if any, or any other concerns.
+
+> #### Do Not Send:
+> - Sensitive or personal information.
+> - Public disclosures of vulnerabilities.
```

Comment:
Is this intended to not have CVE's that are already published from being sent to us for remediation?

Reply:
No, I meant not to publicly share the vulnerability, I see the issues with writing that, I'll remove that line 🙃.